apps/web/src/server/applications/case/examination-timetable/applications-timetable.controller.js

```diff
@@ -79,18 +79,16 @@ export const uniqueTimeTableTypes = {
  */
 export async function viewApplicationsCaseTimetableList(_, response) {
 	const examinationTimetable = await getCaseTimetableItems(response.locals.caseId);
-
-	const timetableItemsViewData = examinationTimetable?.items?.map((timetableItem) =>
-		getTimetableRows(timetableItem)
+	const timetableItemsViewData = examinationTimetable?.items?.map(getTimetableRows) ?? [];
```

Comment:
Do you know how we're currently supporting new timetable items being deleted?

Reply:
Do you mean if you delete a timetable item after the timetable is published?

Reply:
Yeah I was wondering if we could take the same route instead of recording it on a separate record, but I'll resolve this convo because I don't want to hold up the PR - LGTM!